LEAF_Request_Portal/sources/Form.php

```diff
@@ -3900,9 +3900,10 @@ private function isIndicatorOrphan($indicator, &$indicatorList)
      * @param int $recordID
      * @param int $newRecordID
      * @param int $series
-     * @return int 1 for success, errors for failure
+     * @return array|int 1 for success, errors for failure
      */
-    public function copyAttachment($indicatorID, $fileName, $recordID, $newRecordID, $series) {
+    public function copyAttachment($indicatorID, $fileName, $recordID, $newRecordID, $series): array|int
```

Comment:
datatypes for params

Reply:
I know I added some here and there but it really is outside the scope of this ticket.